gateways/guides/linux-install.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Linux Install
+icon: download
+---
+
+# Download the Livepeer binary
+```
+sudo wget https://github.com/livepeer/go-livepeer/releases/download/<RELEASE_VERSION>/livepeer-linux-amd64.tar.gz
+```
+Unpack and remove the compressed file
+```
+sudo tar -zxvf livepeer-linux-amd64.tar.gz
+sudo rm livepeer-linux-amd64.tar.gz
+sudo mv livepeer-linux-amd64/* /usr/local/bin/
+```
+# Generate a new keystore file
+```
+/usr/local/bin/livepeer -network arbitrum-one-mainnet -ethUrl <ARBITRUM RPC URL> -gateway
+exit
+```
+**When generating a new keystore file, the program will prompt you for a password. This password is used to decrypt the keystore file and access the private key. Make sure to never share or lose access to either the password or the keystore file**
```

Comment:
Could maybe be replaced by callout box to make it standout more.

Reply:
DONE